api/net/Areas/Subscriber/Controllers/UserController.cs

```diff
@@ -91,8 +91,8 @@ public IActionResult Update(UserModel model)
         var user = _userService.FindByUsername(username) ?? throw new NotAuthorizedException($"User [{username}] does not exist");
         var isAdmin = user.Roles.Split(',').Contains($"[{ClientRole.Administrator.GetName()}]");
         if (user.Id != model.Id && !isAdmin) throw new NotAuthorizedException("You are not authorized to update this user.");
-        var result = _userService.UpdatePreferences((User)model) ?? throw new NoContentException("Updated did not return the user");
-        return new JsonResult(new UserModel(result));
+        var updateUser = _userService.UpdateUser((User)model) ?? throw new NoContentException("Updated did not return the user");
```

Comment:
We cannot do this it is a security hole.

Reply:
Fixed.